src/tree/CosmosDBAccountProvider.ts

```diff
@@ -28,7 +28,7 @@ export class CosmosDBAccountProvider implements IChildProvider {
     }
 
     public async loadMoreChildren(node: IAzureNode): Promise<IAzureTreeItem[]> {
-        const client = getCosmosDBManagementClient(node.credentials, node.subscriptionId);
+        const client = getCosmosDBManagementClient(node.credentials, node.subscriptionId, node.environment.resourceManagerEndpointUrl);
```

Comment:
Why do we only pass this in sometimes?

Reply:
If it should be passed in always, please change getCosmosDBManagementClient to require the argument.

Reply:
@StephenWeatherford  any updates?

Reply:
I've made some fixes.  Haven't been able to actually test it against a sovereign cloud, but theoretically it should be fine to check in now.  Have you done any testing?

Reply:
@StephenWeatherford yes I had done some testing before, but I realized I did not commit that last change that you fixed 👍 @PrashanthCorp I have tested the current branch and everything works with sovereigns. Let me know if there's anything else you need.